code/graphics/grstub.cpp

```diff
@@ -48,11 +48,6 @@ int gr_stub_save_screen()
 	return 1;
 }
 
-int gr_stub_tcache_set(int bitmap_id, int bitmap_type, float *u_scale, float *v_scale, int tex_unit = 0)
```

Comment:
Nonono, the external tcache API wasn't used anywhere *outside* the rendering code. Obviously, the OpenGL code still needs something to actually use a texture when rendering the game but the code outside of OpenGL doesn't need to know how that's done. That's the great thing about the new material system. All that nasty "low-level" stuff isn't exposed anymore.